src/main/java/frc/robot/commands/TeleopArcadeDriveCommand.java

```diff
@@ -9,48 +9,50 @@
 
 public class TeleopArcadeDriveCommand extends CommandBase {
 
-    DriveSubsystem driveSubsystem;
-
-
-    public TeleopArcadeDriveCommand(DriveSubsystem subsystem) {
-        driveSubsystem = subsystem;
-        addRequirements(subsystem);
-
-    }
-
-    @Override
-    public void initialize() { 
-        driveSubsystem.resetPose();
-    }
-
-    @Override
-    public void execute() {
-        // System.out.println(Robot.robotContainer.getThrottle().getZ());
-        // System.out.println(Robot.robotContainer.getWheel().getX());
-        if (RobotState.isOperatorControl()) {
-            driveSubsystem.arcadeDrive();
-        }
-
-        if (Robot.robotContainer.getCVButton().get()) {
-            new CVDriveCommand(DriveConstants.CAMERA_TO_FRONT_DISTANCE_INCHES, Robot.robotContainer.getDriveSubsystem()).schedule();
-            end(true);
-        }
-        
-        // System.out.println(driveSubsystem.getHeading());
-        // System.out.println(driveSubsystem.getLeftEncoderDistance() + " " + driveSubsystem.getRightEncoderDistance());
-        // System.out.println(driveSubsystem.getWheelSpeeds());
-    
+	DriveSubsystem driveSubsystem;
+
+	public TeleopArcadeDriveCommand(DriveSubsystem subsystem) {
+		driveSubsystem = subsystem;
+		addRequirements(subsystem);
+
+	}
+
+	@Override
+	public void initialize() {
+		driveSubsystem.resetPose();
+	}
+
+	@Override
+	public void execute() {
+		if (RobotState.isOperatorControl()) { // safety check to make sure nothing is accidentally happening in auto mode
+			driveSubsystem.arcadeDrive();
+
+			if (Robot.robotContainer.getCVButton().get()) {
```

Comment:
Are we expecting the CV button to be released or continue being pressed?

Reply:
The robot drives with CV as long as the CV button is held. If the driver releases the CV button early (ex. if there is another robot crossing paths with ours) it will immediately switch back to driver control.